### PR TITLE
Added support to uninstall rhods for operatorhub installation

### PR DESCRIPTION
**Pull request by tarukumar**

### CODE REVIEW

File: ods_ci/tasks/Resources/RHODS_OLM/uninstall/uninstall.robot [hunk 1]
```diff
@@ -23,11 +24,32 @@ Uninstall RHODS In OSD
...
+   Log To Console   ${output}
+
+Uninstall RHODS In Self Managed Cluster For Operatorhub
+   [Documentation]   Uninstall rhods on self-managed cluster for operatorhub installtion
+   ${return_code}    ${output}    Run And Return Rc And Output   oc create configmap delete-self-managed-odh -n redhat-ods-operator
+   Should Be Equal As Integers	${return_code}	 0   msg=Error creation deletion configmap
+   ${return_code}    ${output}    Run And Return Rc And Output   oc label configmap/delete-self-managed-odh api.openshift.com/addon-managed-odh-delete=true -n redhat-ods-operator
+   Should Be Equal As Integers	${return_code}	 0   msg=Error observed while adding label to configmap
+   Verify Project Does Not Exists  redhat-ods-applications
+   ${return_code}    ${output}    Run And Return Rc And Output   oc delete namespace redhat-ods-operator
```
Comment (bdattoma) [inline ods_ci/tasks/Resources/RHODS_OLM/uninstall/uninstall.robot:55]: could we use the same cleanup.sh script like in CLI script?

**Comment by aloganat** [comment]
Have validated this PR from CI and is working fine.
https://<Jenkins URL>/job/delete-cluster/16/console

**Review by aloganat — APPROVED** [review]
LGTM

**Review by mwaykole — APPROVED** [review]

File: ods_ci/tasks/Resources/RHODS_OLM/uninstall/uninstall.robot (hunk 1 above)
Comment (bdattoma) [reply ods_ci/tasks/Resources/RHODS_OLM/uninstall/uninstall.robot:55]: Nono, I'm saying to replace these lines (50-55) with the cleanup.sh script

**Pull request merged**